### DISCUSSION TITLE
Link to search results

### DISCUSSION THREAD

**Discussion by sbeckeriv** [requarks/wiki#6475] [Help / Questions] [answered]
Dearest Reader, 

I have enjoyed wikijs so far! I am wondering if there is a way to send someone a link to search results like:

wiki.example.com/?search=help

Other than sharing the results I would like to make a custom search keyword/engine for firefox. 

thank you for any advice!
Becker

**Comment by NGPixel** [comment, marked as answer]
No but is planned for a future release. See https://requarks.canny.io/wiki/p/search-results-page
